CMakeLists.txt

```diff
@@ -543,7 +543,11 @@ endif()
 
 message(STATUS "Using Lua interpreter: ${LUA_PRG}")
 
-option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" ON)
+if(DEBUG)
+  option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" OFF)
+else()
+  option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" ON)
+endif()
```

Comment:
Is there a way to do this without repeating the option name and description?

Reply:
Another option `ENABLE_LTO` also has its name and description repeated, so I think no.